libs/blocks/marquee/marquee.js

```diff
@@ -147,7 +147,9 @@ export default function init(el) {
   if (firstDivInForeground.classList.contains('media')) el.classList.add('row-reversed');
 
   const size = getBlockSize(el);
-  decorateButtons(text, size === 'large' ? 'button-xl' : 'button-l');
+  decorateButtons(text, (size === 'large')
+  && !(el.classList.contains('standard')
+  || el.classList.contains('compact')) ? 'button-xl' : 'button-l');
```

Comment:
Ternary statements in function params can be confusing, and here there is at least 3 logical checks before the string assignment. Can we break it out to make it more readable, maintainable? Below is a suggestion, but there may be an even batter way to do it. 

```suggestion
  let buttonClass = 'button-l'
  if (size === 'large' && !el.classList.contains('standard') || el.classList.contains('compact)) {
    buttonClass = 'button-xl'
  }
  decorateButtons(text, buttonClass);
```

Reply:
> Ternary statements in function params can be confusing, and here there is at least 3 logical checks before the string assignment. Can we break it out to make it more readable, maintainable? Below is a suggestion, but there may be an even batter way to do it.

Changes as done as requested, thanks!